tests/integration/attendances.test.js

```diff
@@ -69,6 +69,18 @@ describe('ATTENDANCES ROUTES - POST /attendances', () => {
       expect(response.statusCode).toBe(200);
     });
 
+    it('should create attendance if trainee not registered or in another group but company in course (intra_holding)',
```

Comment:
Le "but" est confusant dans ta phrase non ? Que penses-tu de le changer en "and" ?

Reply:
done